test/test_grid.py

```diff
@@ -286,8 +286,8 @@ def test_read_scrip(self):
         xr_grid_u30 = xr.open_dataset(ug_30)
         ux_grid_u30 = ux.Grid(xr_grid_u30)  # tests from ugrid
 
-    def test_build_face_dimension(self):
-        """Tests the construction of the ``Mesh2_face_dimension`` variable."""
+    def test_build_nNodes_per_face(self):
```

Comment:
Can we utilize the `Grid.__from_vert__()` to input our written`face_nodes_connectivity` and then verify if our dimensions calculation for that is accurate?

Reply:
That's a great idea, I'll do that.